doc/whats_new/v1.1.rst

```diff
@@ -112,6 +112,23 @@ Changelog
   multilabel classification.
   :pr:`19689` by :user:`Guillaume Lemaitre <glemaitre>`.
 
+:mod:`sklearn.inspection`
+........................
+
+- |Enhancement| in
+  :meth:`~sklearn.inspection.PartialDependenceDisplay.from_estimator` and
+  :meth:`~sklearn.inspection.PartialDependenceDisplay.from_predictions`, allow
+  `kind` to accept a list of strings to specify for each feature interaction
+  which type of plot a user wants to draw.
+  :pr:`19438` by :user:`Guillaume Lemaitre <glemaitre>`.
+
+- |Fix| in
+  :meth:`~sklearn.inspection.PartialDependenceDisplay.from_estimator` and
+  :meth:`~sklearn.inspection.PartialDependenceDisplay.from_predictions`,
+  instead of raising an error, asking for ICE plots (`kind='both'` or
+  `kind='individual'`) on a 2-way plot will use regular PDP and raise a
+  warning. :pr:`19438` by :user:`Guillaume Lemaitre <glemaitre>`.
```

Comment:
Are tags part of the entries' sentences?

```suggestion
- |Enhancement| In
  :meth:`~sklearn.inspection.PartialDependenceDisplay.from_estimator` and
  :meth:`~sklearn.inspection.PartialDependenceDisplay.from_predictions`, allow
  `kind` to accept a list of strings to specify for each feature interaction
  which type of plot a user wants to draw.
  :pr:`19438` by :user:`Guillaume Lemaitre <glemaitre>`.

- |Fix| In
  :meth:`~sklearn.inspection.PartialDependenceDisplay.from_estimator` and
  :meth:`~sklearn.inspection.PartialDependenceDisplay.from_predictions`,
  instead of raising an error, asking for ICE plots (`kind='both'` or
  `kind='individual'`) on a 2-way plot will use regular PDP and raise a
  warning. :pr:`19438` by :user:`Guillaume Lemaitre <glemaitre>`.
```